sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+sonar.projectKey=zlux:zlux-app-manager
+sonar.projectName=ZLUX Application Manager
+sonar.projectVersion=1.0.0
```

Comment:
What is the meaning of the version? zlux is 0.9.3 but this is 1.0.0. In reality this is a core repo to zlux so I'd expect them to be the same.